README.md

```diff
@@ -1,257 +1,52 @@
+YEAT (**Y**our **E**verday **A**ssembly **T**ool) is an all-in-one platform designed to assemble multiple samples of varying read types (both short and long) using a combination of reliable, widely used, and cutting-edge algorithms such as SPAdes and Flye. It utilizes a Snakemake workflow to preprocess sample reads, perform assembly, and postprocess the resulting contigs in parallel.
 
-## Installation
+| Read Types  | Supported Algorithms |
+| ------------- | ------------- |
+| Paired-end  | SPAdes<sup>[1](#reference-1)</sup>, MEGAHIT<sup>[2](#reference-2)</sup>, Unicycler<sup>[3](#reference-3)</sup>, PenguiN<sup>[4](#reference-4)</sup>, VelvetOptimiser<sup>[5](#reference-5)</sup> |
+| Single-end | SPAdes<sup>[1](#reference-1)</sup>, MEGAHIT<sup>[2](#reference-2)</sup>, Unicycler<sup>[3](#reference-3)</sup>, PenguiN<sup>[4](#reference-4)</sup>, VelvetOptimiser<sup>[5](#reference-5)</sup> |
+| PacBio CLR (<20% error) | Flye<sup>[6](#reference-6)</sup>, Canu<sup>[7](#reference-7)</sup>, Unicycler<sup>[3](#reference-3)</sup> |
+| PacBio Corrected (<3% error) | Flye<sup>[6](#reference-6)</sup>, Canu<sup>[7](#reference-7)</sup>, Unicycler<sup>[3](#reference-3)</sup> |
+| PacBio HiFi (<1% error) | Flye<sup>[6](#reference-6)</sup>, Canu<sup>[7](#reference-7)</sup>, Hifiasm<sup>[8](#reference-8)</sup>, Hifiasm-meta<sup>[9](#reference-9)</sup>, Unicycler<sup>[3](#reference-3)</sup>, metaMDBG<sup>[10](#reference-10)</sup> |
+| ONT Regular (<20% error) | Flye<sup>[6](#reference-6)</sup>, Canu<sup>[7](#reference-7)</sup>, Unicycler<sup>[3](#reference-3)</sup> |
+| ONT Corrected (<3% error) | Flye<sup>[6](#reference-6)</sup>, Canu<sup>[7](#reference-7)</sup>, Unicycler<sup>[3](#reference-3)</sup> |
+| ONT High-quality (<5% error) | Flye<sup>[6](#reference-6)</sup>, Canu<sup>[7](#reference-7)</sup>, Unicycler<sup>[3](#reference-3)</sup> |
+
+## Installation (for Linux only)
 
 ```
 git clone https://github.com/bioforensics/yeat.git
 cd yeat
 conda env create -f environment.yml
+conda create -n yeat-metaMDBG -c bioconda "metamdbg>=1.0" -y
 conda create -n yeat-velvet -c bioconda "perl-velvetoptimiser>=2.2" -y
 conda activate yeat
 pip install .
 ```
 
-### Bandage
-
-In order to run [`Bandage`](https://github.com/rrwick/Bandage) in YEAT's workflow, users will need to manually download the pre-built binaries files [here](https://rrwick.github.io/Bandage/) or build it from the source code [here](https://github.com/rrwick/Bandage). Instructions for building Bandage from the source code can be found [here](https://github.com/rrwick/Bandage#building-from-source).
-
-Once the binary file has been obtained, users will need to update their environment's `$PATH` to include the path to the directory containing the binary file.
+## Usage
 
-Example for Ubuntu:
 ```
-export PATH=~/projects/Bandage:$PATH
+yeat-auto short_reads --files {short_read1} {short_read2} > config.cfg
+yeat --outdir {path} config.cfg
```

Comment:
You might want to make it clear that the user should manually check edit the config file between running `yeat-auto` and `yeat`. And I'd guess that the `--seq-path` option would be the more common configuration route.

```
yeat-auto {sample names} --seq-path {path to dir with Fastq files} > config.cfg
# Check and edit config.cfg before proceeding
yeat --outdir {path} config.cfg
```